.gitignore

```diff
@@ -171,6 +171,7 @@ tmp
 known_hosts
 
 packages/**/globalConfig.json
+packages/forms-web-app/src/public/javascripts
```

Comment:
Is it worth doing this as a wildcard in case we need to do the same for LPA Questionnaire?

Reply:
Done